### DISCUSSION TITLE
Sonoff Zigbee dongle plus E issue with zigbee-herdsman

### DISCUSSION THREAD

**Discussion by vasilykom** [Koenkk/zigbee2mqtt#17062] [Devices]
Hello,
Zigbee2mqtt doesnt work. Tried to update firmware (the latest now)
Once it was possible to run in for 10 seconds.

[13:25:08] INFO: Preparing to start...
[13:25:08] INFO: Socat not enabled
[13:25:08] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-03-17 13:25:10: Logging to console and directory: '/config/zigbee2mqtt/log/2023-03-17.13-25-10' filename: log.txt
Zigbee2MQTT:info  2023-03-17 13:25:10: Starting Zigbee2MQTT version 1.30.2 (commit #unknown)
Zigbee2MQTT:info  2023-03-17 13:25:10: Starting zigbee-herdsman (0.14.96)
Error: Reset error: Error: {"sequence":-1} after 2000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:298:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)

Could you please help?
I am using PC, VirtualBox

**Comment by vasilykom** [comment]
enabled: false
master: pty,raw,echo=0,link=/tmp/ttyZ2M,mode=777
slave: tcp-listen:8485,keepalive,nodelay,reuseaddr,keepidle=1,keepintvl=1,keepcnt=5
options: "-d -d"
log: false

server: mqtt://core-mosquitto:1883
user: mqtt-user
password: admin

port: >-
  /dev/serial/by-id/usb-ITEAD_SONOFF_Zigbee_3.0_USB_Dongle_Plus_V2_20221130092155-if00
adapter: ezsp

**Comment by vasilykom** [comment]
It seems for me that this module has issue, but i have two and same issue with the second unit.

**Comment by Brandycat123** [comment]
I am having the same issue installing Zigbee2MQTT in Home Assistant:
[14:37:45] INFO: Preparing to start...
[14:37:45] INFO: Socat not enabled
[14:37:46] INFO: Starting Zigbee2MQTT...
Zigbee2MQTT:info  2023-03-19 14:37:47: Logging to console and directory: '/config/zigbee2mqtt/log/2023-03-19.14-37-47' filename: log.txt
Zigbee2MQTT:info  2023-03-19 14:37:47: Starting Zigbee2MQTT version 1.30.2 (commit #unknown)
Zigbee2MQTT:info  2023-03-19 14:37:47: Starting zigbee-herdsman (0.14.96)
Error: Reset error: Error: {"sequence":-1} after 2000ms
    at /app/node_modules/zigbee-herdsman/src/adapter/ezsp/driver/uart.ts:298:23
    at Queue.executeNext (/app/node_modules/zigbee-herdsman/src/utils/queue.ts:32:32)
I am new to this and so this is a new build. It is under Windows10 PC running Virtual Box and Home Assistant with a Sonoff 3.0 USB Dongle Plus. Home Assistant  has recognized the dongle and an IKEA Zigbee remote so I assume the dongle is installed correctly. I have flashed it but it made no difference.
I need to use some Samotech SM323 dimmers that ZHA will not recognise.
Some reports say I need to disable ZHA in Home Assistant but I cannot see how this is done.
Please help....Thanks

**Comment by Dranoeleo** [comment]
I had the same Problem when i changed my Coordinator Stick to a Sonoff ZBDongle-P/ZBDongle-E (same issue with both variants).
Solution was:
- uninstall zigbee2mqtt
- uninstall Mosquitto
- disable ZHA
- delete folder /usr/share/hassio/homeassistant/zigbee2mqtt
- delete file /usr/share/hassio/homeassistant/zigbee.db
- delete folder /usr/share/hassio/addons/core/mosquitto
- reinstall mosquitto and zigbee2mqtt

Every settings are lost but i spent so much time trying to fix this issue that it was worth it..
Your file/folder pathes can be different according to your HA installation.
